### DISCUSSION TITLE
Does It Blend (compile SASS)?

### DISCUSSION THREAD

**Discussion by kristianoye** [mickaelistria/eclipse-bluesky#1622] [Q&A] [answered]
So I have World Wide Developer installed and it is definitely integrated with my generic editor, but it is it supposed to compile my SASS files automatically?  I don't even see an option to compile the file.  Is WWD just an editor or does it provide compiler services? 

<img width="791" alt="image" src="https://github.com/eclipse-wildwebdeveloper/wildwebdeveloper/assets/9437527/fbc6e4e3-5a03-4925-8928-d34cb4c5c0ae">

I am using Eclipse 4.32 on Windows 11 with WWD 1.3.6

**Comment by mickaelistria** [comment, marked as answer]
No, the editor just allows edition. To compile sass, you can setup a project builder that invoke `sass`: on a project, right-click > Properties > Builder > New > Program and configure it for `sass` and see in "Build Options" to configure it to run on a change in *.sass files.

**Reply by kristianoye** [reply]
Thanks, sorry I did not give kudos previously.
